lib/modules/datasource/repology/index.ts

```diff
@@ -109,7 +109,7 @@ export class RepologyDatasource extends Datasource {
 
   @cache({
     ttlMinutes: 60,
-    namespace: `datasource-${RepologyDatasource.id}-list`,
+    namespace: `datasource-${RepologyDatasource.id}`,
```

Comment:
doesn't this collide with the other cache decorator?

Reply:
There is no other decorator (see how I change the types and nothing breaks)